src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java

```diff
@@ -229,8 +229,8 @@ int run(String[] args) {
             List<String> remaining = optionsHelper.handleOptions(this, args);
             if (remaining.size() > 0 && !options.suggestProviders) {
                 throw taskHelper.newBadArgs("err.orphan.arguments",
-                                            remaining.stream().collect(Collectors.joining(" ")))
-                                .showUsage(true);
+                        remaining.stream().collect(Collectors.joining(" ")))
```

Comment:
Formatting nit: this change makes it a little hard to see it's an argument passed to `newBadArgs`.